templates/pulpcore-worker@.service.erb

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Pulp Worker
 After=network-online.target
-Wants=network-online.target
+Wants=network-online.target postgresql.service
```

Comment:
Reading this file I debated adding Redis as well, but wasn't quite sure about it.

Reply:
Makes sense (also to foreman and dynflow)